bootstrap.js

```diff
@@ -257,6 +257,10 @@ let prefObserver = {
     case "theme":
     case "thumbs.contain":
     case "thumbs.hidebuttons":
+      enumerateTabs(function(aWindow) {
+        aWindow.newTabTools.updateUI();
+      });
+      break;
```

Comment:
This change doesn't do anything. It's identical to the case below, which is what happens already.